### ISSUE TITLE
[Crash report]  'utf-8' codec can't decode %2

### ISSUE THREAD

**Issue by matheuswillder** [bottlesdevs/Bottles#737]
This crash report was generated by Bottles.

**Details**
* **package**: Flatpak
* **version**: 2021.11.14-treviso-4

**Log**
```python3
'utf-8' codec can't decode byte 0xe7 in position 12: invalid continuation byte  File "/app/share/bottles/bottles/utils.py", line 317, in __target
    result = self.task_func(*args, **kwargs)

  File "/app/share/bottles/bottles/backend/manager.py", line 1089, in create_bottle
    Runner.set_windows(config, config["Windows"])

  File "/app/share/bottles/bottles/backend/runner.py", line 506, in set_windows
    Runner.reg_add(

  File "/app/share/bottles/bottles/backend/runner.py", line 595, in reg_add
    print(Runner.run_command(config, command, comunicate=True))

  File "/app/share/bottles/bottles/backend/runner.py", line 425, in run_command
    return subprocess.Popen(

```
Well, new user here, so please be patient because I still can't master Bottles (although I'm not new to Linux/Wine). Also, sorry my English, it's not my main language.

A few weeks ago I got to know Bottles and installed it to test, and it worked perfectly. After a few days of testing Debian 11 and Bottles, I reinstalled the distribution to be my permanent system and then installed Bottles, but I can't create any Bottle. It stop on "Setting Windows version..." and after several minutes locked the only way out is to close the window. When opening again the log above is displayed.

I know that some new versions have been released recently, but since I wasn't upgrading due to using a system just for testing, I don't know exactly when the problem started. I can't believe I did anything wrong either, I just installed Bottles and tried to use it like I had before, but I'll try to install and test again later on a Live system.

**Comment by mirkobrombin** [comment]
Hi, can you tell me what runners have installed?

**Comment by matheuswillder** [comment]
Hi, thank you for the answer.

> Hi, can you tell me what runners have installed?

Vaniglia, but after a few tries I also installed Proton-GE and tried to create a custom Bottle with it, but the same problem occurs for all of them.

My old computer with integrated graphics unfortunately doesn't support Vulkan but apparently the installer also installs DXVK by default. But like I said, I tried Bottles for a while on a test system before and it worked perfectly.
